crates/aptos-ledger/src/lib.rs

```diff
@@ -0,0 +1,260 @@
+// Copyright © Aptos Foundation
```

Comment:
Given we intend for this to be used as a library, I recommend we add this:
```
#![deny(missing_docs)]
```

This will enforce that all public functions / structs this crate exposes have doc comments.